listView.js

```diff
@@ -96,14 +106,15 @@ class EnhancedListView extends Component {
     }
   }
   render() {
-    return <ListView
-      scrollRenderAheadDistance={RENDER_AHEAD_DISTANCE}
-      scrollEventThrottle={1}
-      {...this.props}
-      onScroll={this.onScroll.bind(this)}
-      ref={component => this.listView = component}
-      renderRow={this.renderRow.bind(this)}
-      renderSectionHeader={this.renderSectionHeader.bind(this)}/>
+    return (
+      <ListView
+        {...this.props}
```

Comment:
The props are ordered very specifically right now - the component's callbacks *must* be used. Each callback checks if the user provided a prop and will call it accordingly.